roles/openshift-prep/tasks/prerequisites.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: "Cleaning yum repositories"
+  command: "yum clean all"
+
+- name: "Install required packages"
+  yum:
+    name: "{{ item }}"
+    state: latest
+  with_items:
+    - wget
+    - git
+    - net-tools
+    - bind-utils
+    - bridge-utils
+    - bash-completion
+    - vim-enhanced
+
+- name: "Update all packages (this can take a very long time)"
```

Comment:
Note that this may require a host reboot - i.e.: if the kernel changed, etc. We  have a small role that does the update and reboot the host if needed (and wait for it to come back). I'd suggest consider something like that for future in a follow-up PR.
https://github.com/redhat-cop/infra-ansible/tree/master/roles/update-host